### ISSUE TITLE
jruby support

### ISSUE THREAD

**Issue by ahorek** [sass/sassc-ruby#119]
there're some failures on jruby
https://travis-ci.org/sass/sassc-ruby/jobs/534281860

it looks like all failures are caused by loading a tempfile in compile_file_context see
https://github.com/sass/libsass/blob/c55821a43551f4a4090fd962cd40443a9d3e54aa/src/context.cpp#L425
it returns an error status 3 - SassC::SyntaxError: Error: File to import not found or unreadable: styles.scss
note that the tempfile exists on the filesystem.

**Comment by bolandrm** [comment]
I don't use jruby, so a contributor will have to submit a patch for this.

**Comment by ahorek** [comment]
I diged into it a little and I've found this function
https://github.com/sass/libsass/blob/b260394263f8db363e0b1bd335edcead011800c3/src/file.cpp#L58

**Comment by ahorek** [comment]
ok, the problem is that Dir.chdir isn't propagated outside of the jvm process, I'm not sure it's even possible.
anyway the problem affects tests only, sassc works fine on jruby.

@bolandrm would you accept a PR that will remove dependency on https://github.com/bhb/test_construct ?

**Comment by bolandrm** [comment]
@ahorek yes, if you need to remove that dependency to resolve the test failures that would be fine.

**Comment by ahorek** [comment]
I've chosen to skip tests that depends on a tempfile. Thanks @bolandrm
